recipes/remove.rb

```diff
@@ -0,0 +1,40 @@
+#
+# Cookbook:: ucarp
+# Recipe:: remove
+#
+# Copyright:: 2018, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_recipe 'ucarp::data_bag'
+
+chef_gem 'chef-rewind'
+require  'chef/rewind'
+
+# Ignore failure from restart notification after service is removed
+rewind 'service[ucarp]' do
+  ignore_failure true
+end
```

Comment:
Please do not use chef-rewind anymore. It's been deprecated in favor of the [edit_resource](https://docs.chef.io/dsl_recipe.html#edit-resource) method.